components/twitter/twitter.app.js

```diff
@@ -20,16 +28,49 @@ module.exports = {
     },
     count: {
       type: "string",
-      label: "Count",
-      description: "The maximum number of tweets to return (up to 100)",
+      label: "Count (advanced)",
+      description: "The maximum number of tweets to return per API request (up to `100`)",
       optional: true,
       default: "100",
     },
+    maxRequests: {
+      type: "string",
+      label: "Max API Requests per Execution (advanced)",
+      description: "The maximum number of API requests to make per execution (e.g., multiple requests are required to retrieve paginated results). **Note:** Twitter [rate limits API requests](https://developer.twitter.com/en/docs/basics/rate-limiting) per 15 minute interval.",
+      optional: true,
+      default: "5",
+    },
+    from: {
+      type: "string",
+      label: "From",
+      description: "The screen name of the user (e.g., `pipedream`)",
```

Comment:
I see `screen_name` is what the api uses but does the user-facing language use this? I've only heard of `handle`.... but what do i know